toolchain/sem_ir/file.h

```diff
@@ -198,7 +199,10 @@ class File : public Printable<File> {
   // TODO: When the AST can be created before creating `File`, initialize the
   // pointer in the constructor and remove this function. This is part of
   // https://github.com/carbon-language/carbon-lang/issues/4666
-  auto set_cpp_ast(clang::ASTUnit* cpp_ast) -> void { cpp_ast_ = cpp_ast; }
+  auto set_cpp_ast(clang::ASTUnit* cpp_ast) -> void;
+  auto cpp_mangle_context() -> clang::MangleContext* {
```

Comment:
```suggestion
  auto clang_mangle_context() -> clang::MangleContext* {
```

Would be more consistent with clang_decls below, but I guess less consistent with cpp_ast above.

Maybe we should just pick one. clang is part of the type (its namespace) so maybe that one wins?

Reply:
Renamed this one.
We should be consistent.
We should also find a way to bridge the gap between the feature (C++ interop) and the implementation (which uses Clang).
I think rename `cpp_ast` to `clang_ast_unit` makes sense, but will defer that to a separate PR.

Reply:
Yeah. I know we discussed this at the summit briefly and we're probably not going to do "C interop", since that creates weird parallel worlds that Carbon has to worry about, but I do consider that it isn't impossible, and we'd still be using all the `clang` things to do that.

Reply:
Yes, makes sense.
For semantics I think we should use Clang when we're referring to how and C++ when we're referring to what.

Reply:
That SGTM

Reply:
Sent out #5926.